### PR TITLE
Document docs/ignore shortcode

### PR DESCRIPTION
**Pull request by jdbaldry**

### CODE REVIEW

**Comment by jdbaldry** [comment]
Converting to draft because of challenges with the `docs/ignore` implementation on the Killercoda side documented in https://github.com/grafana/killercoda/pull/49

File: docs/sources/write/shortcodes/index.md [hunk 1]
```diff
@@ -381,6 +381,36 @@ Produces:
 
 {{< docs/experimental product="experimental-feature" featureFlag="its-feature-flag" >}}
 
+## Docs/ignore
+
+The `docs/ignore` shortcode ignores the content between the start and end markers so it doesn't appear in the rendered webpage.
+
+Use this shortcode when you want to transform source documentation into Killercoda tutorials and want some content to only exist in the tutorial output.
```
Comment (jdbaldry) [inline docs/sources/write/shortcodes/index.md:388]:
```suggestion
Use this shortcode when you want to transform source documentation into Killercoda tutorials and want some content to only exist in the tutorial output.
You should only use ordinary Markdown as the inner content of the shortcode.
```

File: docs/sources/write/shortcodes/index.md [hunk 2]
```diff
@@ -381,6 +381,37 @@ Produces:
...
+You should only use ordinary Markdown as the inner content of the shortcode.
+
+For more information about Killercoda transformation, refer to [About the transformer tool](https://github.com/grafana/killercoda/blob/staging/docs/transformer.md#about-the-transformer-tool).
+
+### Example
+
+```markdown
+This is before the ignore.
```
Comment (chri2547) [inline docs/sources/write/shortcodes/index.md:396]:
```suggestion
This is rendered before the ignore.
```

File: docs/sources/write/shortcodes/index.md [hunk 3]
```diff
@@ -381,6 +381,37 @@ Produces:
...
+
+{{</* docs/ignore */>}}
+This isn't rendered.
+{{</* /docs/ignore */>}}
+
+This is after the ignore.
```
Comment (chri2547) [inline docs/sources/write/shortcodes/index.md:402]:
```suggestion
This is rendered after the ignore.
```

File: docs/sources/write/shortcodes/index.md [hunk 4]
```diff
@@ -381,6 +381,37 @@ Produces:
...
+```
+
+Produces:
+
+This is before the ignore.
```
Comment (chri2547) [inline docs/sources/write/shortcodes/index.md:407]:
```suggestion
This is rendered before the ignore.
```

File: docs/sources/write/shortcodes/index.md [hunk 5]
```diff
@@ -381,6 +381,37 @@ Produces:
...
+
+{{< docs/ignore >}}
+This isn't rendered.
+{{< /docs/ignore >}}
+
+This is after the ignore.
```
Comment (chri2547) [inline docs/sources/write/shortcodes/index.md:413]:
```suggestion
This is rendered after the ignore.
```

**Review by chri2547 — APPROVED** [review]
a few suggestions.

**Review by clayton-cornell — APPROVED** [review]

**Pull request merged**